_config.yml

```diff
@@ -14,7 +14,7 @@ execute:
 
 # Information about where the book exists on the web
 repository:
-  url: https://github.com/napari-staging-site/napari-staging-site.github.io # Online location of your book
+  url: https://https://github.com/napari/napari.github.io # Online location of your book
```

Comment:
looks like you added an extra `https://` here?

Reply:
sure did. thanks!

Reply:
(that makes it _extra_ secure)